modules/cart/src/Plugin/Block/CartBlock.php

```diff
@@ -206,11 +205,25 @@ protected function getCartViews(array $carts) {
 
   /**
    * {@inheritdoc}
-   *
-   * @todo Find proper cache tags to make this cacheable
    */
-  public function getCacheMaxAge() {
-    return 0;
+  public function getCacheContexts() {
+    return Cache::mergeContexts(parent::getCacheContexts(), ['cart']);
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  public function getCacheTags() {
```

Comment:
Also, for what it's worth, I sent a twee to WimLeers to see if he has any input to make sure this is right

https://twitter.com/nmdmatt/status/812119091548917760